DependencyInjection/HttplugExtension.php

```diff
@@ -175,6 +177,9 @@ private function configurePluginByName($name, Definition $definition, array $con
     {
         switch ($name) {
             case 'cache':
+                if (!class_exists(CachePlugin::class)) {
```

Comment:
Should we do that in the `Configuration` class instead to provide a better error message?

Reply:
oh, indeed. with a validate it looks much nicer!